### PR TITLE
add method trackServerInstall

### PR DESCRIPTION
**Pull request by akondratyev**

### CODE REVIEW

File: src/main/java/com/appmetr/s2s/AppMetr.java [hunk 1]
```diff
@@ -73,6 +75,12 @@ public AppMetr(String token, String url) {
         this(token, url, new MemoryBatchPersister());
     }
 
+    public void trackServerInstall(String userId) {
```
Comment (chemist777) [inline src/main/java/com/appmetr/s2s/AppMetr.java]: It should be a separate action like ServerInstallAction, not method.

Comment (akondratyev) [reply src/main/java/com/appmetr/s2s/AppMetr.java]: what's the purpose?
this event isn't customizable, shouldn't have different fields.

if we extract, client will have to 1) create obj 2) call 'track'
here we can just call 'track'

Comment (chemist777) [reply src/main/java/com/appmetr/s2s/AppMetr.java]: Analysts want to pass custom attributes to this event.

Comment (akondratyev) [reply src/main/java/com/appmetr/s2s/AppMetr.java]: is it better now?

File: src/main/java/com/appmetr/s2s/events/ServerInstall.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package com.appmetr.s2s.events;
+
+public class ServerInstall {
```
Comment (chemist777) [inline src/main/java/com/appmetr/s2s/events/ServerInstall.java]:
```suggestion
public class Events {
```

File: src/main/java/com/appmetr/s2s/events/ServerInstall.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+
+    public static Action create() {
```
Comment (chemist777) [inline src/main/java/com/appmetr/s2s/events/ServerInstall.java]:
```suggestion
    public static Event serverInstall(String userId) {
```

File: src/main/java/com/appmetr/s2s/events/Events.java [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package com.appmetr.s2s.events;
+
+public class Events {
+
+    public static Action serverInstall() {
```
Comment (chemist777) [inline src/main/java/com/appmetr/s2s/events/Events.java]: you forgot to add userId parameter.

**Pull request merged**